cicd/pipelines/installing-pipelines.adoc

```diff
@@ -47,6 +47,8 @@ include::modules/op-disabling-automatic-creation-of-rbac-resources.adoc[leveloff
 
 * You can learn more about installing Operators on {product-title} in the xref:../../operators/admin/olm-adding-operators-to-cluster.adoc#olm-adding-operators-to-a-cluster[adding Operators to a cluster] section.
 
+* To install and deploy in-cluster Tekton Hub, see xref:../../cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc#using-tekton-hub-with-openshift-pipelines[Using Tekton Hub with OpenShift Pipelines]. 
```

Comment:
Do you want to use {tekton-hub} or {pipelines-title} for any of the other instances of those words in this PR?